### PR TITLE
规则转换 torch.nn.modules.BatchNorm2d

### PR DESCRIPTION
**Pull request by LokeZhou**
### PR Docs

torch.nn.modules.BatchNorm2d 已有映射文档，同torch.nn.BatchNorm2d
### PR APIs

```bash
torch.nn.modules.BatchNorm2d
```

### CODE REVIEW

**Review by zhwesky2010 — DISMISSED** [review]
LGTM

**Comment by zhwesky2010** [comment]
@LokeZhou 需要处理下冲突

**Review by zhwesky2010 — APPROVED** [review]
LGTM
